### ISSUE TITLE
organization README.md

### ISSUE THREAD

**Issue by gaelforget** [JuliaClimate/meta#16]
Dear org members

as you may have noticed, the landing page on the org is a bit different since I added README.md via the new .github repo earlier today

The initial README.md is essentially of list of links to all org repos with their respective descriptions. Nothing fancy, or controversial imho, but I thought this was already better (& more fair) than just highlighting a few repos by pinning them.

I am hoping to be able to make at least a few cosmetic improvements over the coming week or so

Please consider making suggestions here on what you'd like to see in the org README.md moving forward

**Comment by Datseris** [comment]
Oh NICE! Thanks a lot for the effort @gaelforget ! We can improve on this as time goes by! Although I'd vote that making this much longer isn't much useful because it would start taking too long until a user sees the repositories of the organization!

**Comment by Alexander-Barth** [comment]
I was wondering if we can add a nice plot to the readme and stumbled upon this plot from https://github.com/JuliaClimate/GlobalOceanNotebooks:
![](https://user-images.githubusercontent.com/20276764/135203125-e663713e-48c9-42e0-bbd7-bd4ba222b70a.png)

Are the labels "Nature" and "Human+Nature" are not reversed?

**Comment by Datseris** [comment]
yeah, the labels here are _clearly wrong_. But I don't think this the best plot we can go for. Anything shown a geospatial map of the entire planet is visually much nicer :D

**Comment by gaelforget** [comment]
thanks @Alexander-Barth for pointing out the legend glitch. 

Could you the geospatial map from the same series I guess https://gaelforget.github.io/ClimateModels.jl/dev/examples/IPCC.html 

or maybe make a row with small versions of both plots + the org logo or something like that

**Comment by gaelforget** [comment]
btw, I hope to revisit the stuff in the GlobalOceanNotebooks repo soon. some of it is outdated / redundant + I want to add others

**Comment by gaelforget** [comment]
another thing I was thinking to add is a star button for each repo -- like the one seen @ https://juliaclimate.github.io/GlobalOceanNotebooks/ 

does anyone know of a good way to do this in markdown?

**Comment by Datseris** [comment]
> does anyone know of a good way to do this in markdown?

Exactly what I was wondering! I think this does the trick, right? https://github.com/Naereen/badges#github-stars

**Comment by gaelforget** [comment]
> > does anyone know of a good way to do this in markdown?
> 
> Exactly what I was wondering! I think this does the trick, right? https://github.com/Naereen/badges#github-stars

done

**Comment by gaelforget** [comment]
![IPCC_fig5](https://user-images.githubusercontent.com/20276764/143275851-c165be3b-ca6e-44ab-bcd0-3598c04f2ab6.png)

**Comment by gaelforget** [comment]
![IPCC_fig1b](https://user-images.githubusercontent.com/20276764/143275888-ff02f149-225f-45ac-ae5e-1049e15ab215.png)

**Comment by gaelforget** [comment]
Have now added the two plots discussed above + the star badges

At this point I feel, we could remove the pinned repo part. It's now sort of redundant (all are essentially pinned in README.md) and in the way of the repo list found below in the org landing page.

**Comment by gaelforget** [comment]
![IPCC_fig5_at4C](https://user-images.githubusercontent.com/20276764/143278660-3dc6dbdf-e037-4de8-a976-d0a5a1b48e14.png)

**Comment by gaelforget** [comment]
![IPCC_fig5_at2c](https://user-images.githubusercontent.com/20276764/143279553-41c0a2b7-081f-42f9-b285-c4166b81770e.png)

**Comment by gaelforget** [comment]
wondering about the picture banner(s) : at the top, bottom, or both?

some thoughts : maybe both a top banner and a bottom banner is effective. could vary plots. link to broader picture galleries via https://juliaclimate.org

**Comment by Datseris** [comment]
Last two pictures have same title and colorbar but the actual plot is different...? why?

**Comment by gaelforget** [comment]
> Last two pictures have same title and colorbar but the actual plot is different...? why?

good point. I added legends + a reference to the IPCC atlas

**Comment by Datseris** [comment]
I like a lot the banner the current page has. it's not produced with julia but it is beautiful nevertheless! The plots representing each individual package are made in Julia, so we got that aspect covered! My vote would be to not replace the current banner for a climate plot. But I tend to lean more towards artistic perspectives so we need more opinions!

**Comment by gaelforget** [comment]
> it's not produced with julia but it is beautiful nevertheless!

not sure if that's what you meant but these plots were generated with Makie.jl ( [this notebook](https://raw.githubusercontent.com/gaelforget/ClimateModels.jl/master/examples/IPCC.jl) )

**Comment by Datseris** [comment]
The banner of the homepage I see is:

![image](https://user-images.githubusercontent.com/19669089/143325606-4b406130-3365-4765-915a-9e30321ba0ac.png)

This certainly wasn't produced with Makie.jl. I simply stated that I like this banner a lot, and I vote to not change it for a plot. Perhaps I'm confused about where you suggest that we use the plots...?
